sagemathinc/cocalc
# Add new location to see your mentions

# Description
Currently:
<img width="721" alt="Screen Shot 2019-04-18 at 6 54 26 PM" src="https://user-images.githubusercontent.com/618575/56400964-70e92280-620b-11e9-9f98-22edafe2ad64.png">


UI for no mentions:
<img width="1179" alt="Screen Shot 2019-04-18 at 6 55 04 PM" src="https://user-images.githubusercontent.com/618575/56400974-7e9ea800-620b-11e9-9cee-8f063f956c54.png">


# Testing Steps
Do this with 2 accounts open. One will be the mentioner and the other the mentioned.
The mentioned person should have the notifications page open.

1. Mention with
    1. a very long comment and short comment
    1. the mention is at the beginning, middle, or end.
1. Change the mentioner's name
1. Change the project title
1. Mention them in another project. The mention should appear at the top of the list.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
